app/javascript/components/advanced-settings-form/advanced-settings-form.schema.js

```diff
@@ -0,0 +1,22 @@
+import { validatorTypes } from '@@ddf';
+
+const createSchema = () => ({
+  fields: [
+    {
+      component: 'code-editor',
+      id: 'settings',
+      name: 'settings',
+      mode: 'yaml',
+      label: '',
+      hideLabel: true,
+      // modes: ['yaml'],
+      // isRequired: true,
+      validate: [
+        { type: validatorTypes.REQUIRED },
+        // custom YAML format validator
```

Comment:
is this a  TODO?

Reply:
Kinda, still thinking about that...right now we don't have anything like this.